### PR TITLE
Add `font-display: swap`

### LINKED ISSUE #69: Add `font-display`
On Google's [Web Fundamentals - Web Font Optimization page](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization#customize_the_text_rendering_delay), they recommend using `font-display`:

`font-display: auto; /* or block, swap, fallback, optional */`

> To ensure consistency moving forward, the CSS Working Group has proposed a new @font-face descriptor, [font-display](https://drafts.csswg.org/css-fonts-4/#font-display-desc), and a corresponding property for controlling how a downloadable font renders before it is loaded.

Thanks for a great tool!

### PR DESCRIPTION
**Pull request by tomayac**
Fixes #69.

This PR adds `font-display: swap;`, with a pointer to read the MDN docs for other options.

### CODE REVIEW

**Review by souvik362 — APPROVED** [review]
Thanks @tomayac for making these changes,
waiting for maintainers to merge this patch to the `master` branch.

**Comment by tomayac** [comment]
Any signal from the maintainer, @majodev?

**Comment by jasperf** [comment]
Any news on this issue? Realized the fonts we pull in do not have `font-display:swap` and would love to have this loaded in the generated CSS on the fly.

**Comment by Buildbetter** [comment]
Yes it important to use   font-display: swap; - please add it to generator its a simple step.

**Comment by hebbet** [comment]
> Yes it important to use font-display: swap; - please add it to generator its a simple step.

try that one:
https://nextgenthemes.com/google-webfont-downloader/

**Pull request merged**
